test/regressions/index.test.js

```diff
@@ -26,6 +28,15 @@ async function main() {
     }
   });
 
+  let errorConsole;
+  let errorConsoleCounter = 0;
+
+  page.on('console', async (msg) => {
+    if (msg.args().length > 0 && (msg.type() === 'error' || msg.type() === 'warning')) {
```

Comment:
Why the `length` check? I'd argue that this either doesn't happen or we should fail regardless.  

Otherwise why is `console.error()` ok?

Reply:
The callback gets called multiple times with a length === 0. I believe it's related to network requests that return 404, like the images.

Reply:
No, actually, it's about `Failed to load resource: net::ERR_FAILED`.

Reply:
Thanks for digging!